sdcm/cluster.py

```diff
@@ -1320,12 +1320,18 @@ def _read_system_log_and_publish_events(
                         if not exclude:
                             LOGGER.debug(line)
                 match = backtrace_regex.search(line)
+                one_line_backtrace = []
                 if match and backtraces:
                     data = match.groupdict()
                     if data['other_bt']:
                         backtraces[-1]['backtrace'] += [data['other_bt'].strip()]
                     if data['scylla_bt']:
                         backtraces[-1]['backtrace'] += [data['scylla_bt'].strip()]
+                elif "backtrace:" in line.lower() and "0x" in line:
```

Comment:
It's a common practice to add a comment with an example of a line which parsed here. Actually, this advice included in our Best Practices by @bentsi

Reply:
Added